### ISSUE TITLE
The operation couldn’t be completed. (LibFido2Swift.FIDO2Error error 0.)

### ISSUE THREAD

**Issue by molind** [XcodesOrg/XcodesApp#643]
**Describe the bug**
Trying to login into my dev account using hardware key. It shows field and says insert your pysical key and enter the PIN. I have YubiKey 5C NFC. There is no PIN, but it requires to touch the key to authenticate.  When I touch the key it enters some hidden text into that PIN field and next screen is an error `The operation couldn’t be completed. (LibFido2Swift.FIDO2Error error 0.)`

**To Reproduce**
Tap install.
Sign in with hardware key.
Xcodes shows PIN screen.
You touch the key.
Xcodes shows an error.

There is no way to skip PIN screen.

**Expected behavior**
I should be able to sign in.

**Screenshots**
<img width="1012" alt="Screenshot 2024-10-30 at 12 26 08" src="https://github.com/user-attachments/assets/3f8c6082-c50c-4202-bf2c-1c653bbfe502">

**Version**
 - OS: macOS 15.1
 - Xcodes: 2.4.0

**Comment by MattKiazyk** [comment]
@kinoroy can I get you to perhaps answer any issues on the hardware key?

**Comment by kinoroy** [comment]
Yeah at the moment, devices without a FIDO2 PIN set aren't supported, sorry about that. 

However, it seems I incorrectly assumed a PIN was required by Apple as the Relying Party but that doesn't seem to be the case now that I am testing it.

So, it should be possible to support PIN-less authentication and I can work on opening a PR to allow it

Thanks for the report!
